addons/web/static/src/less/navbar.less

```diff
@@ -176,8 +176,8 @@
             }
 
             &.o_menu_systray {
-                .o-position-absolute(0px, @odoo-navbar-height - 2, auto, @odoo-navbar-height - 2);
-                height: @odoo-navbar-height - 2;
+                .o-position-absolute(0px, @odoo-navbar-height, auto, @odoo-navbar-height);
+                height: @odoo-navbar-height;
```

Comment:
Note: `@odoo-navbar-height - 2` is used at some other places (not only navbar.less). If you can check that too, it could be great. I can do it if you prefer.